src/workerd/api/cache-operations.js

```diff
@@ -0,0 +1,306 @@
+// Copyright (c) 2017-2023 Cloudflare, Inc.
```

Comment:
Nit: The file name does not indicate that this is part of a test, should be renamed